Update Roundcube to 1.6.0

kiekerjan opened the pull request:
Roundcube 1.6.0 provides support for php 8.1 (see https://github.com/roundcube/roundcubemail/releases/tag/1.6.0)
Also updated the plugins and the reference to the Owncloud address book.

stvhay commented:
Hmm. I'll merge this on my fork and see how it goes.

stvhay commented:
Sorry if there is a better way, since this is not my PR:

```
diff --git a/setup/webmail.sh b/setup/webmail.sh
index ef02dff..c05425c 100755
--- a/setup/webmail.sh
+++ b/setup/webmail.sh
@@ -115,8 +115,8 @@ cat > $RCM_CONFIG <<EOF;
 \$config['log_dir'] = '/var/log/roundcubemail/';
 \$config['temp_dir'] = '/var/tmp/roundcubemail/';
 \$config['db_dsnw'] = 'sqlite:///$STORAGE_ROOT/mail/roundcube/roundcube.sqlite?mode=0640';
-\$config['default_host'] = 'ssl://localhost';
-\$config['default_port'] = 993;
+\$config['imap_host'] = 'ssl://localhost';
+\$config['imap_port'] = 993;
 \$config['imap_conn_options'] = array(
   'ssl'         => array(
      'verify_peer'  => false,
@@ -124,7 +124,7 @@ cat > $RCM_CONFIG <<EOF;
    ),
  );
 \$config['imap_timeout'] = 15;
-\$config['smtp_server'] = 'tls://127.0.0.1';
+\$config['smtp_host'] = 'tls://127.0.0.1';
 \$config['smtp_conn_options'] = array(
   'ssl'         => array(
      'verify_peer'  => false,
```

kiekerjan commented:
> Some of the configuration options need to be updated per https://roundcube.net/news/2022/07/28/roundcube-1.6.0-released.

Never saw that ☹️  Did seem to work though. Anyway:
- Changed default_host to imap_host: `$config['imap_host'] = 'ssl://localhost:993';`
- Changed smtp_server to smtp_host: `$config['smtp_host'] = 'tls://127.0.0.1';`
- Removed default_port (included 993 in imap_host)
